dev/com.ibm.ws.jaxrs.2.1_fat_extended/.classpath

```diff
@@ -1,14 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry kind="src" path="fat/src"/>
-	<classpathentry kind="src" path="test-applications/subResourceApp/src"/>
```

Comment:
any reason these have been reordered? not a big deal

Reply:
I put them alphabetic order.  I think it helps in finding stuff in Eclipse easier.

Reply:
agreed